pkgs/by-name/ne/netatalk/package.nix

```diff
@@ -60,10 +66,12 @@ stdenv.mkDerivation (finalAttrs: {
     dbus
     docbook-xsl-nons
     libxslt
```

Comment:
These are not needed anymore for netatalk 4.2.0.

Instead, please add `cmark` (or `cmark-gfm`) to this list. This is what we use now for generating roff man pages from Markdown.